cmd/orb-server/startcmd/params.go

```diff
@@ -60,6 +61,12 @@ const (
 	vctURLFlagUsage = "Verifiable credential transparency URL."
 	vctURLEnvKey    = "ORB_VCT_URL"
 
+	vctMonitoringIntervalFlagName  = "vct-monitoring-interval"
+	vctMonitoringIntervalEnvKey    = "VCT_MONITORING_INTERVAL"
+	vctMonitoringIntervalFlagUsage = "The interval in which VCTs are monitored to ensure that proofs are anchored. " +
```

Comment:
Is this flag's value in seconds or does it accept a time unit as well, similar to other time parameters? If so - consider adding that `"For example, '30s' for a 30 second timeout."` that helps make it more clear.

Reply:
(I see below that it is indeed a duration - my point was that the documentation is a bit inconsistent)

Reply:
The usage description specifies 10s as the default, but I guess I could provide an example. I'll do it in a subsequent PR since there are other flags that have the same shortcoming. I think we should make all time durations use the same standard (e.g. 1s, 1m, etc.). There are certain parameters that assume milliseconds or seconds.